### PR TITLE
feat(tutorial): add cheatsheets to py lists&dicts MTA-3350

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: tutorials/python-lists-dicts/index.mdx [hunk 1]
```diff
@@ -458,6 +460,16 @@ The table below provides a summary of everything covered above:
 | Get the lowest item                       | `min(class_marks)`                                            |
 | Get the highest item                      | `max(class_marks)`                                            |
 
+### Lists cheatsheet
+
+<Lightbox src="python-lists-cheatsheet.webp" alt="" size="large" />
+
+<Message type="tip">
+  This content is also available as a printable PDF file. Click the link below to download:
```
Comment (nerda-codes) [inline tutorials/python-lists-dicts/index.mdx:468]:
```suggestion
  This content is also available as a printable PDF file. [Download the Python lists cheatsheet - Printable](python-lists-cheatsheet.pdf)
```

File: tutorials/python-lists-dicts/index.mdx [hunk 2]
```diff
@@ -458,6 +460,16 @@ The table below provides a summary of everything covered above:
...
+
+    - [Python lists cheatsheet - Printable](python-lists-cheatsheet.pdf)
```
Comment (nerda-codes) [inline tutorials/python-lists-dicts/index.mdx:470]:
```suggestion
```

File: tutorials/python-lists-dicts/index.mdx [hunk 3]
```diff
@@ -696,9 +710,18 @@ The table below provides a summary of everything covered above:
...
 | Delete a key/value pair from the dictionary | `del student_marks['Aneesa']`                                |
 | Clear the whole dictionary (delete all key/value pairs) | `student_marks.clear()`                          |
 | Get the length of the dictionary  | `len(student_marks)`                                                   |
 
+### Dictionaries cheatsheet
+
+<Lightbox src="python-dicts-cheatsheet.webp" alt="" size="large" />
+
+<Message type="tip">
+  This content is also available as a printable PDF file. Click the link below to download:
```
Comment (nerda-codes) [inline tutorials/python-lists-dicts/index.mdx:724]:
```suggestion
  This content is also available as a printable PDF file. [Download the Python lists cheatsheet - Printable](python-lists-cheatsheet.pdf)
```

File: tutorials/python-lists-dicts/index.mdx [hunk 4]
```diff
@@ -696,9 +710,18 @@ The table below provides a summary of everything covered above:
...
+
+    - [Python dictionaries cheatsheet - Printable](python-dicts-cheatsheet.pdf)
```
Comment (nerda-codes) [inline tutorials/python-lists-dicts/index.mdx:726]:
```suggestion
```

**Review by nerda-codes — COMMENTED** [review]
A few suggestions :)

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**